src/System.Text.Json/tests/Serialization/TestClasses.SimpleTestClassWithObject.cs

```diff
@@ -141,6 +143,8 @@ public void Initialize()
             MyStringICollectionT = new string[] { "Hello" };
             MyStringIReadOnlyCollectionT = new string[] { "Hello" };
             MyStringIReadOnlyListT = new string[] { "Hello" };
+
+            MyStringToStringDict = new Dictionary<string, string> { { "key", "value" } };
```

Comment:
Additional testing thoughts:
- `IDictionary<,>` and well as `Dictionary<,>` meaning we need to create the dictionary instance when we see the interface only (like we do for `IList<T>` for example).
- Returning values directly without being a property: `IDictionary<string, int> d = JsonSerializer.Parse(@"{""foo"":42}");
- Serializing of polymorphic `object` (`Dictionary<string, object>`) like we do for properties. We don't support deserialize yet for `object`
- Dictionary containing an enumerable or another dictionary (not that important for now)

Reply:
> IDictionary<,>

YES please 😆 - it would be nice to include `IReadOnlyDictionary<>` it's easy.